package.json

```diff
@@ -2,6 +2,7 @@
   "name": "rdmo",
   "scripts": {
     "build:prod": "webpack --config webpack.config.js --mode production",
```

Comment:
I would suggest that `build:prod` fails on warnings and `testing` does not, but contains the performance hints.